docs/content/en/integration/openid-connect/oauth-2.0-bearer-token-usage.md

```diff
@@ -0,0 +1,277 @@
+---
+title: "OAuth 2.0 Bearer Token Usage"
+description: "An introduction into utilizing the Authelia OAuth 2.0 Provider as an authorization method"
+lead: "An introduction into utilizing the Authelia OAuth 2.0 Provider as an authorization method."
+date: 2023-12-29T20:38:06+11:00
+draft: false
+images: []
+menu:
+  integration:
+    parent: "openid-connect"
+weight: 611
+toc: true
+---
+
+Access Tokens can be granted which can be leveraged as bearer tokens for the purpose of authorization in place of the
+Session Cookie Forwarded Authorization Flow. This is performed leveraging the
+[RFC6750: OAuth 2.0 Bearer Token Usage](https://datatracker.ietf.org/doc/html/rfc6750) specification.
+
+## Authorization Endpoints
+
+When utilizing the `authelia.bearer.authz` scope clients are able to request users grant access to a bearer token which
+can be utilized in place of the standard authorization flows for protected resources by utilizing
+[OAuth 2.0 Bearer Token Usage] authorization scheme norms (i.e. using the bearer scheme) when specifically accessing
+resources protected via the forwarded authentication flow which directly integrates into proxies. It should be clearly
+noted that this means it is not for accessing
+the Authelia API.
```

Comment:
I find this paragraph really hard to understand. Specifically the first sentence is very long and chains together a lot of ideas.  
Here's an attempt at splitting it up and rewording a bit:
```suggestion
When utilizing the `authelia.bearer.authz` scope, clients are able to request users create and grant them access to a bearer token.
This token can then be used following [OAuth 2.0 Bearer Token Usage] norms (i.e. the bearer scheme) to access resources protected via the forwarded authentication flow (i.e. `access_control` rules). It should be clearly
noted that such a token does not grant access to the Authelia API.
```